src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -329,10 +329,10 @@ function MoneyRequestConfirmationList({
             taxCode = transaction?.taxCode ?? TransactionUtils.getDefaultTaxCode(policy, transaction) ?? '';
         }
         const taxPercentage = TransactionUtils.getTaxValue(policy, transaction, taxCode) ?? '';
-        const taxAmount = TransactionUtils.calculateTaxAmount(taxPercentage, taxableAmount);
+        const taxAmount = TransactionUtils.calculateTaxAmount(taxPercentage, taxableAmount, currency);
```

Comment:
@ShridharGoel 

Why did we use `currency` and not `transcation.currency`? `currency` seems to be the currency associated to the mileage rate or the policy: `(mileageRate as MileageRate)?.currency ?? policyCurrency;`. What if the transaction is not a distance transaction and it has a different currency from the policy default?

Reply:
No worries, I'll update it here: https://github.com/Expensify/App/pull/43519